packages/desktop-client/src/components/settings/index.js

```diff
@@ -128,7 +128,17 @@ function Settings({
         marginInline: globalPrefs.floatingSidebar && !isMobile() ? 'auto' : 0,
       }}
     >
-      <Page title="Settings">
+      <Page
+        title="Settings"
+        titleStyle={
+          isMobile()
+            ? {
+                backgroundColor: colors.n11,
```

Comment:
Whatever the current value is, let's match the theme color for this page down at the bottom of the file:
https://github.com/actualbudget/actual/blob/a46a9d86a06c1c8e0f74cd316907e5b342fce9ec/packages/desktop-client/src/components/settings/index.js#L186

Also, I believe the HOC I added for theme color could be removed if everything's a `Page` that can accept `style` (separate effort).

Reply:
Good idea: https://github.com/actualbudget/actual/pull/887/commits/a99aaf4df93ad6d18b0b98f8f6210b12efe3ed5f